ceph_deploy/mon.py

```diff
@@ -257,7 +261,7 @@ def mon_create(args):
         try:
             # TODO add_bootstrap_peer_hint
             LOG.debug('detecting platform for host %s ...', name)
-            distro = hosts.get(host, username=args.username)
+            distro = hosts.get(host, username=args.username, callbacks=[packages.ceph_is_installed])
```

Comment:
This looks like the only `hosts.get()` call that didn't get split into multiple lines...  Not sure I care. :)